static/app/components/charts/optionSelector.tsx

```diff
@@ -102,6 +108,27 @@ class OptionSelector extends Component<Props, State> {
   }
 }
 
+const TruncatedLabel = styled('span')`
+  max-width: 400px;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
```

Comment:
we have this [style](https://github.com/getsentry/sentry/blob/master/static/app/styles/overflowEllipsis.tsx) that you can just import

Reply:
dang. TIL, will switch :+1: